labs/a7.md

```diff
@@ -109,16 +109,19 @@ To submit the lab, answer the questions on Gradescope.
 
 ## NFS
 
-We have provided a NFS server for you to connect to at `staff` with two
-different directories, one read-only and one read-write. First install the
-`nfs-common` package so that you can mount directories over NFS. Then, use the
-`mount` command (remember to look at the `man` pages or search online if you do
-not recognize a command) to mount from `staff:/opt/lab7/public` (the remote
-directory) to your local directory at `/opt/lab7/read-only`. Once you do this,
+We have provided a NFS server for you to connect to at `staff4.decal.xcf.sh` with two
+different directories, one read-only and one read-write. 
+
+First, **install the `nfs-common` package** so that you can mount directories over NFS. 
+Then, **use the `mount` command** (remember to look at the `man` pages or search online if you do
+not recognize a command) to mount from `staff4.decal.xcf.sh:/opt/lab7/public` (the remote
+directory) to your local directory at `/mnt/`. Once you do this,
 you should see a file with a secret inside it in `/opt/lab7/read-only`. You can
```

Comment:
This path needs changing too, it'll be within `/mnt` instead, same with the path on line 133.